backends-clickhouse/src/main/scala/io/glutenproject/execution/CHHashAggregateExecTransformer.scala

```diff
@@ -389,4 +395,26 @@ case class CHHashAggregateExecTransformer(
       newChild: SparkPlan): CHHashAggregateExecTransformer = {
     copy(child = newChild)
   }
+
+  override protected def getAdvancedExtension(
+      validation: Boolean = false,
+      originalInputAttributes: Seq[Attribute] = Seq.empty): AdvancedExtensionNode = {
+    val enhancement = if (validation) {
+      // Use a extension node to send the input types through Substrait plan for validation.
+      val inputTypeNodeList = originalInputAttributes
+        .map(attr => ConverterUtils.getTypeNode(attr.dataType, attr.nullable))
+        .asJava
+      Any.pack(TypeBuilder.makeStruct(false, inputTypeNodeList).toProtobuf)
+    } else {
+      null
+    }
+    val optimizationContent = s"isStreaming=$isStreaming\n" +
```

Comment:
why needs this `isStreaming` parameter

Reply:
keep it from hash base transform

Reply:
but this `isStreaming` is different from the one in the `HashAggregateExecBaseTransformer`